api/api_test.go

```diff
@@ -240,6 +246,32 @@ func TestCreateDeleteNetwork(t *testing.T) {
 	if errRsp != &successResponse {
 		t.Fatalf("Unexepected failure: %v", errRsp)
 	}
+
+	// Create with labels
+	labels = map[string]interface{}{
+		netlabel.EnableIPv6:                                  "true",
+		"com.docker.network.bridgename":                      "abc",
```

Comment:
Yes it should. It won't matter for this test itself, but we yes, label is wrong. Will fix it. Thanks.